src/pyuvdata/uvbeam/cst_beam.py

```diff
@@ -209,11 +209,11 @@ def read_cst_beam(
         self.pixel_coordinate_system = "az_za"
         self._set_cs_params()
 
-        out_file = open(filename, "r")
-        line = out_file.readline().strip()  # Get the first line
-        out_file.close()
+        with open(filename) as out_file:
+            line = out_file.readline().strip()  # Get the first line
+            out_file.close()
```

Comment:
this isn't strictly necessary inside the context manager now.

Reply:
specifically the close call

Reply:
ah, yes. I'll take it out